CAM/SetUseHiDof.md

```diff
@@ -8,4 +8,5 @@ ns: CAM
 void SET_USE_HI_DOF();
 ```
 
-
+Needs to be called every tick to make the active camera use a high depth of field.  
+The DoF can be customized using [SET_CAM_NEAR_DOF](_#_0x3FA4BF0A7AB7DE2C), [SET_CAM_FAR_DOF](_#_0xEDD91296CD01AEE0), [SET_CAM_USE_SHALLOW_DOF_MODE](_#_0x16A96863A17552BB), [SET_CAM_DOF_STRENGTH](_#_0x5EE29B4D7D5DF897) and other DoF related natives. (Example: https://forum.cfx.re/t/camera-depth-of-field-settings/2433316/3)
```

Comment:
I'm not sure what happened here, native references are often written like this:

```
[`_GET_VEHICLE_HORN_HASH`](#_0xACB5DCCA1EC76840)
```

Take this [line](https://github.com/citizenfx/natives/blob/fdb8fccd026770491f80abd19f48d84cbb7f9fb6/AUDIO/OverrideVehHorn.md?plain=1#L14) as an example.

Please double check, thank you!